### ISSUE TITLE
No haptic feedback when hold-to-initiate-paste action for text input

### ISSUE THREAD

**Issue by sethladd** [flutter/flutter#9223]
I noticed that on Android, if you tap-to-hold to initiate a text paste, you'll trigger a haptic feedback when the PASTE menu opens.

In Flutter, though, no haptic feedback trigger when you tap-to-hold.

**Comment by Hixie** [comment]
@sethladd Can you check if this is still broken? I thought we added haptic feedback for Posse a while back.

**Comment by sethladd** [comment]
sorry, I don't notice haptic feedback when long-pressing to trigger selection in Flutter.

**Comment by lesnitsky** [comment]
@Hixie This one could probably be closed, haptic feedback is there (tested on Pixel 2XL)

**Comment by TahaTesser** [comment]
No feedback on Samsung Galaxy M30

<details> 
<summary>flutter doctor -v</summary> 

``` 
[✓] Flutter (Channel dev, 1.19.0-3.0.pre, on Mac OS X 10.15.5 19F96, locale
    en-GB)
    • Flutter version 1.19.0-3.0.pre at /Users/tahatesser/Code/flutter_dev
    • Framework revision 6135091de9 (31 hours ago), 2020-06-01 17:17:03 -0700
    • Engine revision e39301f23f
    • Dart version 2.9.0 (build 2.9.0-11.0.dev 6489a0c68d)

 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at /Users/tahatesser/Code/sdk
    • Platform android-29, build-tools 29.0.3
    • ANDROID_HOME = /Users/tahatesser/Code/sdk
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.5)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.5, Build version 11E608c
    • CocoaPods version 1.9.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.0)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 45.1.2
    • Dart plugin version 193.7361
    • Java version OpenJDK Runtime Environment (build
      1.8.0_242-release-1644-b3-6222593)

[✓] VS Code (version 1.45.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.11.0

[✓] Connected device (4 available)
    • iPhone 11  • 0EC80516-8F99-4A0C-89ED-2273579862F9 • ios            •
      com.apple.CoreSimulator.SimRuntime.iOS-13-5 (simulator)
    • macOS      • macOS                                • darwin-x64     • Mac
      OS X 10.15.5 19F96
    • Web Server • web-server                           • web-javascript •
      Flutter Tools
    • Chrome     • chrome                               • web-javascript •
      Google Chrome 83.0.4103.61

• No issues found!
``` 

</details>

**Comment by creativecreatorormaybenot** [comment]
## Issue resolved

I wanted to fix this issue (as part of #89117). During my investigation into the current behavior, I found out that this is indeed already fixed, i.e. the same haptic feedback for selecting text is triggered for **opening the paste menu** _as well_ (the point of this issue).

@TahaTesser if you are not experiencing it, I assume you are also not getting it for selecting text, but then I would assume it has to do with your setup / with `Feedback.forLongPress` in general (but the issue is not about that).

### Investigation

I was able to confirm that https://github.com/flutter/flutter/blob/2512ecd9f66c573cc91cb59248610ec41876f3ff/packages/flutter/lib/src/material/text_field.dart#L121 is called equally for both selecting text and the long press to open paste menu action.

This also means that https://github.com/flutter/flutter/blob/2512ecd9f66c573cc91cb59248610ec41876f3ff/packages/flutter/lib/src/material/text_field.dart#L136 is called for both.

You can see this in action in the following screen recording:


https://user-images.githubusercontent.com/19204050/131235995-974be14b-2354-4fa9-ae3b-f7422431b648.mp4

---

@TahaTesser @sethladd this can be closed 🥳

**Comment by TahaTesser** [comment]
@creativecreatorormaybenot 
Thanks for the investigation, appreciate it 

Closing as fixed. If you disagree, please write in the comments and I will reopen it.
Thank you
